tests/e2e/modules/compliments_spec.js

```diff
@@ -0,0 +1,58 @@
+const Application = require("spectron").Application;
+const path = require("path");
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
+const moment = require("../../../vendor/moment/moment-with-locales.js");
+
+var electronPath = path.join(__dirname, "../../../", "node_modules", ".bin", "electron");
+
+if (process.platform === "win32") {
+	electronPath += ".cmd";
+}
+
+var appPath = path.join(__dirname, "../../../js/electron.js");
+
+var app = new Application({
+	path: electronPath,
+	args: [appPath]
+});
+
+global.before(function () {
+	chai.should();
+	chai.use(chaiAsPromised);
+});
+
+describe("Compliments module", function () {
+	this.timeout(10000);
+
+	describe("parts of days", function() {
+		before(function() {
+			// Set config sample for use in test
+			process.env.MM_CONFIG_FILE = "tests/configs/modules/compliments/compliments_parts_day.js";
+		});
+
+		beforeEach(function (done) {
+			app.start().then(function() { done(); } );
+		});
+
+		afterEach(function (done) {
+			app.stop().then(function() { done(); });
+		});
+
+		it("shows correct compliments for part of day", function () {
+
+			var hour = moment().hour();
```

Comment:
@fewieden Why dont use momentjs?

You mean,  create  3 test case and check if is the part of day accord reflect output module content